tests/python_functional/src/syslog_ng_config/statements/scl/scl.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2015-2018 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+class SCL(object):
+    group_type = "scl"
+    driver_name = "scl"
+    scl_name = ""
+    type = ""
+
+    def __init__(self, driver_type, scl_name):
+        self.type = driver_type
+        self.scl_name = scl_name
+
+class CCHash(SCL):
+    driver_name = "scl"
+    def __init__(self):
+        super(CCHash, self).__init__("rewrite", "credit-card-hash();")
+
+class CCMask(SCL):
```

Comment:
Please feel free to use longer names here (like `CreditCardMask`).

Reply:
This has been addressed by Balazs.